### PR TITLE
CEDS-1051 Supplementary declaration content update

### PR DESCRIPTION
**Pull request by PauloPortugal**

### CODE REVIEW

File: app/views/declaration/office_of_exit.scala.html [hunk 1]
```diff
@@ -30,11 +30,13 @@
...
         @components.error_summary(form.errors)
 
+        @components.section_header(messages("supplementary.summary.locations.header"))
+
         @components.page_title(Some("supplementary.officeOfExit.title"))
 
         @components.input_text(
             field = form("officeId"),
-            label = messages("supplementary.officeOfExit"),
+            label = "",
```
Comment (PatrykRudnicki) [inline app/views/declaration/office_of_exit.scala.html]: Just to be sure. Do you want to remove label?

File: app/views/declaration/supervising_office.scala.html [hunk 2]
```diff
@@ -30,11 +30,13 @@
...
         @components.error_summary(form.errors)
 
+        @components.section_header(messages("supplementary.summary.locations.header"))
+
         @components.page_title(Some("supplementary.supervisingCustomsOffice.title"))
 
         @components.input_text(
             field = form("supervisingCustomsOffice"),
-            label = messages("supplementary.supervisingCustomsOffice"),
+            label = "",
```
Comment (PatrykRudnicki) [inline app/views/declaration/supervising_office.scala.html]: The same question here

File: app/views/declaration/office_of_exit.scala.html (hunk 1 above)
Comment (PauloPortugal) [reply app/views/declaration/office_of_exit.scala.html]: Yes. We have a hint, but no longer a label

File: app/views/declaration/supervising_office.scala.html (hunk 2 above)
Comment (PauloPortugal) [reply app/views/declaration/supervising_office.scala.html]: Yes same thing here. No label, but we have a hint.

**Pull request merged**